index.bs

```diff
@@ -2547,11 +2547,13 @@ Candidate Amendments</h4>
 	If there is no group chartered to maintain a [=technical report=],
 	the [=Team=] <em class="rfc2119">may</em> maintain its [=errata=]
 	and associated [=candidate corrections=].
-	When doing so,
-	it <em class=rfc2119>must</em> be explicit that those [=errata=] and [=candidate corrections=]
-	were written by the [=Team=].
+	Such corrections <em class=rfc2119>must</em> be marked
+	as <dfn>Team correction</dfn>,
+	and are considered non-normative
+	(i.e. not covered)
+	for the purposes of the Patent Policy.
```

Comment:
Perhaps we can link more clearly to the patent policy, and say that the corrections do not constitute a normative portion of the recommendation (cf Patent Policy 8.2 point 2). I would prefer this as otherwise the presence of normative terms (must, should etc.) would appear to contradict this provision and leave us in a grey area.

Something like:

"Such corrections <em class=rfc2119>must</em> be marked as a <dfn>Team correction</dfn>, and as not constituting a normative portion of the Recommendation, as defined in the Patent Policy (i.e. as not covered by the Patent Policy)."

Reply:
s/and as not constituting/and do not constitute/, but otherwise I agree with your suggestion.